spanner/api/Spanner.Samples/Singer.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_delete_data]
+// [START spanner_insert_data]
+// [START spanner_write_data_for_struct_queries]
+// [START spanner_query_with_parameter]
+// [START spanner_quickstart
+
+public class Singer
```

Comment:
Same, let's consider moving this to the samples that need it as inner classes of the sample class.